src/olympia/versions/models.py

```diff
@@ -693,6 +693,18 @@ def can_be_disabled_and_deleted(self):
         return True
 
 
+class VersionReviewerFlags(ModelBase):
+    version = models.OneToOneField(
+        Version, primary_key=True, on_delete=models.CASCADE
+    )
+    needs_human_review_by_mad = models.BooleanField(default=False)
+    pending_rejection = models.DateTimeField(default=None, null=True)
```

Comment:
Nothing is actually using this property currently - that's expected?

Reply:
Yes, just adding the model to unblock other work that will take more time to finish.